src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -179,12 +200,13 @@ public static class TextCatalog
             => new WordEmbeddingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(), modelKind, columns);
 
         /// <summary>
-        /// Tokenizes incoming text in <paramref name="inputColumnName"/>, using <paramref name="separators"/> as separators,
-        /// and outputs the tokens as <paramref name="outputColumnName"/>.
+        /// Create a <see cref="WordTokenizingEstimator"/>, which tokenizes input text using <paramref name="separators"/> as separators.
         /// </summary>
         /// <param name="catalog">The text-related transform's catalog.</param>
-        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
-        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// This column's data type will be a vector of text.</param>
```

Comment:
>vector [](start = 46, length = 6)

unknown size vector

Reply:
Is variable-sized fine?

---
In reply to: [276870655](https://github.com/dotnet/machinelearning/pull/3418#discussion_r276870655) [](ancestors = 276870655)

Reply:
variable-sized (as discussed in the chat)

---
In reply to: [277070083](https://github.com/dotnet/machinelearning/pull/3418#discussion_r277070083) [](ancestors = 277070083,276870655)